src/migrations/r-migration-commands/m129_130_enable_gyro_default_false.ts

```diff
@@ -0,0 +1,18 @@
+import { RecordNode, r, rtp, RT } from "../../r";
+import { IOrder } from "../IOrder";
+
+class Migration implements IOrder {
+  execute (projectJson: unknown) {
+    const pJson = projectJson as RecordNode<RT.project>;
+    const projectF = r.record(pJson);
+
+    if (projectF.get(rtp.project.enable_gyro) === undefined) {
```

Comment:
This will break for existing projects. Many projects rely on `enable_gyro` default value to be `true`. So the users never change this property. 

After this migration, existing projects where enable_gyro = undefined will stop working.